patches/sourcelink/0001-Exclude-test-projects.patch

```diff
@@ -0,0 +1,116 @@
+From 4a051438d29bd96ede45069031462522313ed1bc Mon Sep 17 00:00:00 2001
+From: adaggarwal <aditya.aggarwal@microsoft.com>
+Date: Tue, 2 Jul 2019 16:43:39 +0000
+Subject: [PATCH] Exclude test projects.
+Note: This patch is necessary in fixing arcade #3051 and awaiting uptake. 
```

Comment:
The phrasing makes me think the patch is needed to fix something in Arcade, but it's kind of the other way around. Maybe:

```suggestion
Note: This patch is necessary to work around https://github.com/dotnet/arcade/pull/3051. Removing this patch is awaiting uptake of a new Arcade version.
```

Reply:
Thanks for the suggestion.